app/models/user/permissions.rb

```diff
@@ -89,22 +88,34 @@ def services_member_permission
   end
 
   def has_access_to_service?(service)
-    services_permission = services_member_permission
-    services_permission && services_permission.has_service?(service) || has_access_to_all_services?
+    services_member_permission&.has_service?(service) || has_access_to_all_services?
```

Comment:
The above code refactoring seems like unintended. Original code reads that in case `services_member_permission` is present, then we return `has_service?(service)`, otherwise fall back to `has_access_to_all_services?`. 

With the updated code, `has_access_to_all_services?` will be returned even in case `services_member_permission` is present but `service` is not part of the list.

Did you intend to change original logic or just wanted to update style?

btw maybe we need to rename this `has_access_to_all_services?` method as it is confusing to read.

From your earlier explanation, my understanding is that 
> `has_access_to_all_services?` calls `!admin_sections.include?(:services)`

Which applies to both - users without any access and users with full access.

I think the method can be removed and directly use `!admin_sections.include?(:services)`.